### DISCUSSION TITLE
Help Needed: API Access to Supabase Services Not Enabled

### DISCUSSION THREAD

**Discussion by guilhermegiorgi** [supabase/supabase#29368] [Questions] [answered]
Hi everyone,

I'm trying to access Supabase services via the API, but the dashboard shows that access is not enabled, specifically for PostgREST and Auth. The database and other services seem fine. I've attached a screenshot of the issue for reference.

Has anyone experienced this before or know how to resolve it? Any guidance would be greatly appreciated!

Thank you!
![image](https://github.com/user-attachments/assets/4be0510a-d63b-4fe5-a374-a8df6cfe25a0)

**Comment by GaryAustin1** [comment, marked as answer]
If this is hosted please contact support.
